app/views/categories/index.html.erb

```diff
@@ -0,0 +1,12 @@
+<%= stylesheet_link_tag 'categories', media: 'all', 'data-turbolinks-track': 'reload' %>
+
+<div class ="categories">
+  <%= link_to "New category", new_category_path, class: "link_to newcat" %>
+  <% if @categories.empty? %>
+      <p>No categories available</p>
+        <% else %>
+    <% @categories.each do |category| %>
+          <%= link_to render(category), category_path(category), class: "link_to cats" %>
```

Comment:
- [ ] As requested, please increase the size of the image of the category to match the given design.
![image](https://github.com/tjay1760/Budget-App/assets/91486791/1d9687cd-b973-4349-9ed1-fb07d63ef805)